src/components-examples/cdk/stepper/cdk-linear-stepper-with-form/example-custom-linear-stepper.html

```diff
@@ -0,0 +1,19 @@
+<section class="example-container">
+  <header>
+    <h2>Step {{selectedIndex + 1}}/{{steps.length}}</h2>
+  </header>
+
+  <div [ngTemplateOutlet]="selected ? selected.content : null"></div>
+
+  <footer class="example-step-navigation-bar">
+    <button class="example-nav-button" cdkStepperPrevious>&larr;</button>
+    <button
+      [ngClass]="selectedIndex === i ? 'example-active' : 'example-step'"
```

Comment:
Similar to Kristiyan's comment, this would be the more idiomatic thing here:
```html
<button class="example-step" [class.example-active]="selectedIndex === i"
```
You may have to tweak `.example-step` and `.example-active` to make that work